Design theory

Hotdogdavid opened the pull request.

Hotdogdavid committed:
added the three visitors

Hotdogdavid committed:
getting my message across

Hotdogdavid committed:
Update README.md

sarmenti7 commented on README.md:
```diff
@@ -1,44 +1,15 @@
 
+## Design
+I plan on getting my message across by showcasing my resume next to pictures of my project to show what I have done. I also want to add pictures of the anime I like and information about my personality.
 
```
For your ## Accessibility section, describe how your site will be accessible to users with different impairments? What web accessibility testing will you run to ensure this?

Hotdogdavid committed:
yo

Hotdogdavid committed:
Finsihed the 20 slides
	renamed:    _static/download28.jpeg -> _static/img/download28.jpeg
	renamed:    _static/downloadtiger.jpeg -> _static/img/downloadtiger.jpeg
	modified:   _static/play_pause.html
	modified:   _templates/layout.html
	modified:   conf.py
	modified:   index.md
	modified:   pechakucha.md
